docs/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -189,6 +189,28 @@ enables you to filter anomaly results for one or more selected influencers.
 The query bar suggests influencer field names and values for the selected jobs 
 as you type.
 
+[float]
+=== Simplified creation of {ml} jobs for sample data sets and {metricbeat} data
+
+If you use the {kib} sample data sets, there are now {ml} jobs for the
+sample eCommerce orders and the sample web logs. These jobs demonstrate a
+variety of ways that you can detect anomalous behavior in the sample data. You
+can add the jobs when you <<add-sample-data,load the sample data>> or use the
+supplied configuration when you are creating jobs in the Machine Learning app.
+
+Likewise, if you use the
+{metricbeat-ref}/metricbeat-module-system.html[{metricbeat} system module] to
+monitor your servers, there is a {ml} wizard that recognizes this type of data.
+For example, you can select a supplied configuration to detect unusual increases
```

Comment:
This can make it sound like you can pick and choose which jobs to run from the metricbeat module. Maybe something like 'For example, with the supplied configuration you can detect unusual increases in disk utilization:` ?

Reply:
Good point, done!